examples/tests/passtest.py

```diff
@@ -8,6 +8,8 @@ class PassTest(Test):
 
     """
     Example test that passes.
+
+    :avocado: tags=fast
```

Comment:
most of the `failure_expected` tests are also `fast`, anyway it's not needed to tag them...